ext/phar/tests/gh18953/autoload.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+
+spl_autoload_register(function ($class) {
+    $base_dir = __DIR__ . '/src/';
+
+    $file = $base_dir . str_replace('\\', '/', $class) . '.inc';
+    if (file_exists($file)) {
+        require $file;
+    }
+});
```

Comment:
Can you not move this into the base test file?

Reply:
Not sure `require` matters, so I splited them